src/esw/cameras.py

```diff
@@ -292,8 +288,6 @@ def handle_req(self, req: ChangeCamerasRequest) -> ChangeCamerasResponse:
         :return: A corresponding response.
         """
 
-        # Get most up-to-date available cameras
```

Comment:
Did you think that this was unnecessary? I mean I guess it is updating every second so perhaps

Reply:
Yeah.